target/linux/ramips/dts/mt7628an_hiwifi_hc5661a.dts

```diff
@@ -1,127 +1,46 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
```

Comment:
Oops...
To specify a license for an existing dts, you'd need to ask for an Acked-by from every one who contributed to the original file :(
I think you can drop this line for now and use a separated patch for this task later.

Reply:
@dengqf6 Would you mind if I merge this PR with this line dropped?